tests/outputs/TestPar.cpp

```diff
@@ -58,7 +59,7 @@ TEST(TestPar, write) {
                                                                      },
                                                                      4., 40., 44., 440.)};
 
-  dfl::outputs::Par parWriter(dfl::outputs::Par::ParDefinition(basename, outputDir + "/" + filename, generators));
+  dfl::outputs::Par parWriter(dfl::outputs::Par::ParDefinition(basename, dirname, outputDir + "/" + filename, generators));
```

Comment:
small question: explicit separator like here "/" will work on OS platforms?

Reply:
I don't know but it is true that we can protect ourselves in that case. I'll do the correction. But we'll still have the issue in the config.json if we have multi-directories output dir